app/models/ui/ErrorCallout.scala

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
I am not sure about the names of these two classes. "ErrorCallout" shows its origin in Ocelot. I be tempted to use ErrorMsg for ErrortCallout because it is just a container for an error message and then use something like KeyedErrorMsg for the class linking the error message to an element on the Html page.

Reply:
ErrorMsg is the real UI element going through to the template and contains an id from UI derived FormError. The ErrorCallout is a placeholder for a Callout of type Error which may or may not be turned into an ErrorMsg. The idea being when Questions are built, they will always contain the ErrorCallout objects, however ErroMsgs are only created from the ErrorCallouts if relevant errors are found in the FormData.